paddle/fluid/inference/tests/api/CMakeLists.txt

```diff
@@ -134,6 +134,20 @@ inference_analysis_api_test_with_fake_data(test_analyzer_googlenet
 inference_analysis_api_test_with_fake_data(test_analyzer_resnet50
   "${INFERENCE_DEMO_INSTALL_DIR}/resnet50" analyzer_resnet50_tester.cc "resnet50_model.tar.gz" SERIAL)
 
+#resnet50 int8
+if(WITH_MKLDNN)
+  set(INT8_RESNET50_TARGET_NAME test_analyzer_int8_resnet50)
+  set(INT8_RESNET50_INSTALL_DIR "${INFERENCE_DEMO_INSTALL_DIR}/int8_resnet50")
+  if (NOT EXISTS ${INT8_RESNET50_INSTALL_DIR})
+    inference_download_and_uncompress(${INT8_RESNET50_INSTALL_DIR} "http://paddle-inference-dist.cdn.bcebos.com/int8" "resnet50_int8v2_data.txt.tar.gz")
+    inference_download_and_uncompress(${INT8_RESNET50_INSTALL_DIR} "http://paddle-inference-dist.cdn.bcebos.com/int8" "resnet50_int8_model.tar.gz" )
+  endif()
+  inference_analysis_test(${INT8_RESNET50_TARGET_NAME} SRCS analyzer_int8_resnet50_tester.cc
+    EXTRA_DEPS ${INFERENCE_EXTRA_DEPS} benchmark
+    ARGS --infer_model=${INT8_RESNET50_INSTALL_DIR}/model --infer_data=${INT8_RESNET50_INSTALL_DIR}/data.txt SERIAL)
+  set_tests_properties(${INT8_RESNET50_TARGET_NAME} PROPERTIES TIMEOUT 1200)
```

Comment:
`TIMEOUT 1200` needs 20 minutes?

Reply:
Once I ran randomly into timeout problem and I found somewhere default is 600 so i doubled it. I didn't realize it was in seconds 😆

Reply:
i will leave it to default